tools/test/toolchains/test_toolchains.py

```diff
@@ -8,9 +8,10 @@
 import os
 import sys
 from unittest import TestCase
-
 import mock
 
+UNSUPPORTED_OPTION_SMALL_EXECPTION_STRING = 'small C library option not supported for this target.'
```

Comment:
Define this only in one place so you test wont break if the text changes. Define it somewhere in the tools code and export the variable in the test.